src/ChangeStream.php

```diff
@@ -48,16 +52,27 @@ public function __construct(Cursor $cursor, callable $resumeCallable)
         $this->key = 0;
     }
 
+    /**
+     * @see http://php.net/iterator.current
+     * @return mixed
+     */
     public function current()
     {
         return $this->csIt->current();
     }
 
-    public function getId()
+    /**
+     * @return MongoDB\Driver\CursorId
```

Comment:
Related to my previous comment, this FQCN would be needed since CursorId isn't imported via a `use` statement.